environments/common/inventory/group_vars/all/proxy.yml

```diff
@@ -0,0 +1,2 @@
+# default proxy address to first squid api address port 3128 if squid group non-empty, else empty string to avoid breaking hostvars
+proxy_http_proxy: "{{ 'http://' + hostvars[groups['squid'].0].api_address + ':3128' if groups['squid'] else '' }}"
```

Comment:
Can we make this depend on the `squid_http_port` variable somehow? Or at least add a note somewhere to highlight that this needs to match.

Reply:
Fixed via commit